docs/iris/src/whatsnew/latest.rst

```diff
@@ -219,6 +219,11 @@ This document explains the changes made to Iris for this release
   user guide to clarify how ``Units`` are handled during cube arithmetic.
   (:pull:`3803`)
 
+* `@tkknight <https://github.com/tkknight>`_ overhauled the
+  :ref:`developers_guide` including information on getting involved in
+  becoming a contributor and general structure of the guide.  This resolves
+  :issue:`2170`, :issue:`2331`, :issue:`3453`. (:pull:`3852`)
+
```

Comment:
I'm thinking such an important change should be listed somewhere more prominent than this.  Or are we planning some other way to flag that "it's now easier to get involved"?

Reply:
It does seem that when cutting the release, some of the best features ought to be picked out in the same way as has been done previously (e.g. [v2.3](https://scitools-iris.readthedocs.io/en/latest/whatsnew/2.3.html)). It's tricky since that will need detailed knowledge of **all** the highlighted features!

Reply:
I think that we can bank this as is.

Once the `whatsnew` is complete, if we want to showcase specific entries, then we can do that.